<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color:rgb(173,216,230);
}</string>
  </property>
  <widget class="QSlider" name="progressSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>771</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal
{
   height:7px; 
   width:463px;
   background:gray; 
   border-radius:2px;
}
QSlider::handle:horizontal
{
   background:rgb(52,50,64);
   width:10px;
   height:10px;
   margin:-7px -7px;
   border-radius:5px;
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:20px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:40px;
   max-width:40px;
   min-height:40px;
   max-height:40px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel
{
   border-color:rgb(255,255,255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>SONG NAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:40px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:80px;
   max-width:80px;
   min-height:80px;
   max-height:80px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>reproducir.png</normaloff>reproducir.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pause">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:30px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:60px;
   max-width:60px;
   min-height:60px;
   max-height:60px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>550</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:30px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:60px;
   max-width:60px;
   min-height:60px;
   max-height:60px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_seekback">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:20px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:40px;
   max-width:40px;
   min-height:40px;
   max-height:40px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_mute">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>570</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:20px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:40px;
   max-width:40px;
   min-height:40px;
   max-height:40px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="seek">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>560</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
   border:none;
   border-radius:20px;
   background-color:rgb(255,255,255);
   color:rgb(52,51,64);
   min-width:40px;
   max-width:40px;
   min-height:40px;
   max-height:40px;
   padding:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="vol">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>580</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal
{
   height:7px; 
   width:60px;
   background:gray; 
   border-radius:2px;
}
QSlider::handle:horizontal
{
   background:rgb(52,50,64);
   width:10px;
   height:10px;
   margin:-7px -7px;
   border-radius:5px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>550</y>
     <width>20</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>540</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="screen" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>521</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <zorder>label_name</zorder>
  <zorder>progressSlider</zorder>
  <zorder>open</zorder>
  <zorder>label_time</zorder>
  <zorder>pushButton_play</zorder>
  <zorder>pushButton_pause</zorder>
  <zorder>pushButton_stop</zorder>
  <zorder>pushButton_seekback</zorder>
  <zorder>pushButton_mute</zorder>
  <zorder>seek</zorder>
  <zorder>vol</zorder>
  <zorder>line_3</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>screen</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
